src/bindings/js/node/include/model_wrap.hpp

```diff
@@ -109,6 +109,13 @@ class ModelWrap : public Napi::ObjectWrap<ModelWrap> {
      */
     Napi::Value get_output_shape(const Napi::CallbackInfo& info);
 
+    /**
+     * @brief Function to access operations of the model
```

Comment:
Just use info from python api docs https://docs.openvino.ai/2024/api/ie_python_api/_autosummary/openvino.Model.html#openvino.Model.get_ops
```suggestion
     * @brief Return ops used in the model.
```